src/resources/views/quiz.blade.php

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<html>
+    <head>
+    <title>{{$links->name}}</title> 
+    <link rel="stylesheet" href="{{ asset('reset.css') }}">
+    <link rel="stylesheet" href="{{ asset('style.css') }}">
+    <style></style>
+    </head>
+    <body>
+    <div class="question">
+    <!-- 設問のループ -->
+        @foreach ($questions as $question)
+            <h1 class="title">{{$loop->iteration}}．この地名は何て読む？</h1>
+            <div class="quiz-img-container">
+                <img src="../../img/{{$question->image}}" alt="">
+            </div>
+        <!-- 選択肢のループ -->
+                    <ul id ="quiz-choices{{$question->id}}" class="choices-list">
+                @foreach ($choices[$loop->index] as $choice)
+                        <li id="choice{{$question->id}}_{{$loop->iteration}}"
+                            onclick="clickfunction({{$question->id}},{{$loop->iteration}},{{$choice->valid}})">
```

Comment:
phase1, phase2の名残だと思いますが、onclick属性を使わずにdocument.querySelectorAllとdata-XXなどを組み合わせた使い方ができるとJavaScriptとの棲み分けができるので良いと思います
```suggestion
                            onclick="clickfunction({{$question->id}},{{$loop->iteration}},{{$choice->valid}})">
```